app/partials/project-card.cjsx

```diff
@@ -0,0 +1,46 @@
+React = require('react')
+FlexibleLink = require './flexible-link'
+Translate = require 'react-translate-component'
+
+ProjectCard = React.createClass
+  displayName: 'ProjectCard'
+  propTypes:
+    project: React.PropTypes.object.isRequired
+
+  getDefaultProps: ->
+    avatar: ''
+    name: ''
+    slug: ''
+
+  componentDidMount: ->
+    card = @refs.projectCard
+
+    Promise.resolve (if @props.project.avatar_src then "//#{ @props.project.avatar_src }" else '/assets/simple-avatar.jpg')
```

Comment:
I am not sure. This is pre-existing code I just moved across from `partials/owned-card.cjsx`. I can investigate, though this question shouldn't necessarily delay this PR.